src/main/java/uk/gov/hmcts/reform/professionalapi/controller/SuperController.java

```diff
@@ -134,7 +140,7 @@
             organisationCreationRequestValidator.validateCompanyNumber(organisationCreationRequest);
         }
 
-        if (StringUtils.isBlank(organisationCreationRequest.getSraRegulated())) {
+        if (isBlank(organisationCreationRequest.getSraRegulated())) {
             organisationCreationRequest.setSraRegulated(SRA_REGULATED_FALSE);
```

Comment:
two if conditions are same  i think we can remove one.
isBlank(organisationCreationRequest.getSraRegulated())

Reply:
Sure can be removed. Its production code, can be refactored in next release after QA regression.

Reply:
Created tech debt for this

Reply:
https://tools.hmcts.net/jira/browse/RDCC-1680